dhall/src/Dhall/Main.hs

```diff
@@ -113,13 +116,21 @@ import qualified Text.Pretty.Simple
 
 -- | Top-level program options
 data Options = Options
-    { mode    :: Mode
-    , explain :: Bool
-    , plain   :: Bool
-    , ascii   :: Bool
-    , censor  :: Censor
+    { mode             :: Mode
+    , optionsExplain   :: Maybe Bool
```

Comment:
If you enable the `DuplicateRecordFields` extension then you can keep the original field names